ui/page/send/page.go

```diff
@@ -58,6 +60,7 @@ type Page struct {
 	backdrop  *widget.Clickable
 
 	isFetchingExchangeRate bool
+	isModalLayout          bool
```

Comment:
I'll move this close to the line where you have `modal *cryptomaterial.Modal`. They are related.